packages/web/src/containers/incentivized-pool-card-list-container/IncentivizedPoolCardListContainer.tsx

```diff
@@ -35,12 +35,11 @@ const IncentivizedPoolCardListContainer: React.FC = () => {
   const [page, setPage] = useState(1);
   const router = useRouter();
   const [mobile, setMobile] = useState(false);
-  const { incentivizedPools, isFetchedPools, updatePools, loading } = usePoolData();
+  const { incentivizedPools, isFetchedPools, updatePools, loading: loadingPool } = usePoolData();
```

Comment:
What about `isLoading` or `isLoadingPool` instead of `loadingPool`?

Reply:
i will fix this